pkg/service/replay/replay.go

```diff
@@ -164,8 +164,14 @@ func (r *Replayer) Start(ctx context.Context) error {
 
 			if err != nil {
 				r.config.Test.SkipCoverage = true
+			} else if r.config.Test.Language == models.Javascript && r.config.Test.SkipPreview {
+				r.config.Command = strings.Replace(r.config.Command, "nyc --clean=$CLEAN ", "nyc --clean=$CLEAN --reporter=none ", 1)
 			}
+
 		}
+	}
```

Comment:
instead of replacing you can use the pass a flag in preprocess and directly append there.

Reply:
yes something of that sort.